fix/epsilon/ell1gen.fix.d

```diff
@@ -207,6 +208,8 @@ void main(string[] args)
                 "info|i", "Show heap usage information.", &info,
                 "verbose|v", "Print verbose parser error messages.", &verbose,
                 "write|w", "Toggle default for writing output.", &write,
+                "trailingTokens|t", "allowing arbitrary trailing content before EOF (Oberon2 like behavior)", 
```

Comment:
This is an elegant way of hiding the LL-specific option from the rest of the compiler generator.
However, the option is more like "generate a compiler with an Oberon-style parser" than "ignore trailing content for this specific input".

I don't see, how this option can be made a compiler-generator option in an elegant way. So, I would recommend no option for this behavior at all. Just remove the condition from `recognizeEof`.

Reply:
@linkrope, just to have no misunderstanding between us: 
The option was added to the generated compiler, not the generator! 
The first time, I read your note I thought, you are mixing up generated compiler and generator. Now, I'm not sure whether I maybe get our note wrongly...

Reply:
Not relevant anymore, we go for a backward incompatible change. See below.